app/controllers/lead_providers/report_schools_controller.rb

```diff
@@ -27,6 +27,7 @@ def check_delivery_partner
 
     def success
       @confirm_schools_form = ConfirmSchoolsForm.new(session.delete(:confirm_schools_form))
+      session.delete(:delivery_partner_id)
```

Comment:
Do we want to delete `partnership_csv_upload_id` from session too?

Reply:
That's deleted earlier, in confirm